exercises/space-age/.meta/generator/test_template.erb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative '<%= exercise_name %>'
+
+# Common test data version: <%= abbreviated_commit_hash %>
+class <%= exercise_name_camel %>Test < Minitest::Test
+  DELTA = 0.01
```

Comment:
I wonder if a comment explaining what delta is used for would be helpful for people who are meeting `assert_in_delta` for the first time.